src/backend/aspen/app/app.py

```diff
@@ -20,6 +21,17 @@
     aspen_config = ProductionConfig()
 else:
     aspen_config = DockerComposeConfig()
+
+# We should be able to allow this in all environments and only alert on prod.
+# Init as early as possible to catch more
+sentry_sdk.init(
+    aspen_config.SENTRY_URL,
+    # Set traces_sample_rate to 1.0 to capture 100%
+    # of transactions for performance monitoring.
+    # We recommend adjusting this value in production.
+    traces_sample_rate=1.0,
+)
```

Comment:
based on the comment should we reduce the sample rate? You could also probably remove the `# We recommend adjusting this value in production.` comment.

Reply:
Right now we are only catching exceptions and AFAIK we shouldn't experience too many of them. And events seem to be grouped. But it could get overwhelming. Any suggestions for a good value? We might actually need to let Sentry run in prod for a week or so to get an idea of what to adjust it to.

Reply:
maybe 0.7?